core-kotlin-modules/core-kotlin-lang-oop-4/src/test/kotlin/ExtensionFieldsAlternativesUnitTest.kt

```diff
@@ -0,0 +1,117 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+
+class ExtensionFieldsAlternativesUnitTest {
+
+    private fun String.toTitleCase(): String = this.split(" ").joinToString(" ") { it.capitalize() }
```

Comment:
Did you apply the Baeldung formatter to the code? I'd expect to see line breaks for chained method calls